independent-publisher-connectors/Workable/apiDefinition.swagger.json

```diff
@@ -0,0 +1,1692 @@
+                            "title": "Social profile username"
+                          },
+                          "url": {
+                            "type": "string",
+                            "description": "Social profile url",
+                            "title": "Social profile url"
+                          }
+                        }
+                      },
+                      "description": "Social profiles",
+                      "title": "Social profiles"
+                    },
+                    "tags": {
+                      "type": "array",
+                      "items": {},
+                      "description": "Tags",
+                      "title": "Tags"
+                    },
+                    "location": {
+                      "type": "object",
+                      "properties": {
+                        "location_str": {
+                          "type": "string",
+                          "description": "Location",
+                          "title": "Location"
+                        },
+                        "country": {
+                          "type": "string",
+                          "description": "Country",
+                          "title": "Country"
+                        },
+                        "country_code": {
+                          "type": "string",
+                          "description": "Country code",
+                          "title": "Country code"
+                        },
+                        "region": {
+                          "type": "string",
+                          "description": "Region",
+                          "title": "Region"
+                        },
+                        "region_code": {
+                          "type": "string",
+                          "description": "Region code",
+                          "title": "Region code"
+                        },
+                        "city": {
+                          "type": "string",
+                          "description": "City",
+                          "title": "City"
+                        },
+                        "zip_code": {
+                          "type": "string",
+                          "description": "Zip code",
+                          "title": "Zip code"
+                        }
+                      },
+                      "description": "Location",
+                      "title": "Location"
+                    },
+                    "originating_candidate_id": {
+                      "type": "string",
+                      "description": "Originating candidate ID",
+                      "title": "Originating candidate ID"
+                    }
+                  },
+                  "description": "Candidate",
+                  "title": "Candidate"
+                }
+              }
+            }
+          }
+        },
+        "summary": "Get Candidates by ID",
+        "operationId": "Candidates_id",
```

Comment:
@DavidBreathe please use Camel case on all operationsIds. for example here should be "CandidatesId". Please see [here](https://learn.microsoft.com/en-gb/connectors/custom-connectors/connector-string-guidance#operations)
please fix all operationIds in the swagger file.